test/Integration/Porter/PorterTest.php

```diff
@@ -121,11 +121,38 @@ public function testImport()
 
         self::assertInstanceOf(PorterRecords::class, $records);
         self::assertNotSame($this->specification, $records->getSpecification());
+        self::assertInstanceOf(CountableProviderRecords::class, $records->getPreviousCollection());
+        self::assertSame('foo', $records->current());
+    }
+
+    public function testNonCountableIteratorImport()
+    {
+        $porter = (new Porter)->registerProvider(
+            $this->provider =
+                \Mockery::mock(Provider::class)
+                    ->shouldReceive('fetch')
+                    ->andReturn($this->iterateOne())
+                    ->byDefault()
```

Comment:
During `setup()` we specify the return value is a default so it may be overridden in tests. Since this is the override we are writing it does not serve any purpose to declare this is to be default also. This `byDefault()` declaration can simply be removed.